src/import-v1/index.ts

```diff
@@ -333,6 +340,70 @@ async function* readSubdomains(importDir: string) {
   }
 }
 
+class StxVestingTransform extends stream.Transform {
+  isVesting: boolean = false;
+  constructor() {
+    super({ objectMode: true, highWaterMark: SUBDOMAIN_BATCH_SIZE });
+  }
+
+  _transform(data: string, _encoding: string, callback: stream.TransformCallback) {
+    if (data.startsWith('-----END')) {
+      this.isVesting = false;
+    }
+
+    if (this.isVesting) {
+      const parts = data.split(',');
+      if (parts[0] !== 'address') {
+        const tokenOfferingLocked: DbTokenOfferingLocked = {
+          address: parts[0],
```

Comment:
This `address` string can be a b58 (bitcoin) address or a STX address. It needs to be converted to a STX address if it's b58. Similar to what was done here: https://github.com/blockstack/stacks-blockchain-api/pull/542

Reply:
Updated this address part.